app/src/main/java/com/globant/example/mentorapp/home/presentation/view/fragment/ListUsersViewFragment.java

```diff
@@ -59,7 +59,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         staggeredGridLayoutManager = new StaggeredGridLayoutManager(
                 getResources().getInteger(R.integer.number_of_users_columns), LinearLayoutManager.VERTICAL);
         if (model.getUsers() != null || model.getUsers().getValue() != null) {
-            listUsersAdapter = new ListUsersAdapter(model.getUsers().getValue());
+            listUsersAdapter = new ListUsersAdapter(model.getUsers().getValue(), new OnUserClickListener());
```

Comment:
Define just one new instance of the ListUserAdapter, if the data must be refreshed you can pass it through a setter method and notify the adapter.